src/tests/run_load_tests.sh

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-docker build --cache-from smartsdk/quantumleap -t smartsdk/quantumleap ../../
```

Comment:
looks like something went wrong here. this change was **not** brought into master by #363---you deleted it before merging. not sure why, perhaps you merged master into this branch instead of rebasing? even so this delete shouldn't be there. git weirdness. let's not waste time on this, we'll discard this change when rebasing on master after #347 gets merged.

Reply:
rebase was not clean, there were conflicts to fix, so things may have got weird

Reply:
no prob, like I said we can fix this on the next rebase :-)

Reply:
put it back, see 78976b5